### DISCUSSION TITLE
Support for batch with return_exceptions=True in RemoteRunnable

### DISCUSSION THREAD

**Discussion by rahermur** [langchain-ai/langserve#657] [General]
I noticed a comment on the RemoteRunnable class in LangServe mentioning that batch with return_exceptions=True is not supported due to the lack of exception translation from the server:

```
    The following features are not supported:

    - `batch` with `return_exceptions=True` since we do not support exception
      translation from the server.
```

I was planning to use RemoteRunnable in batch mode, but when an exception occurs on the server, the entire batch is lost. I was wondering if this feature is on the roadmap for future development?

**Comment by nadeems2024** [comment]
Do you have any work around for this issue ?
